dashboard/application/config/rt_config.yml

```diff
@@ -0,0 +1,5 @@
+server: "http://rt:34000"
+user: "root"
+pass: "password"
+queues:
+  - "General"
```

Comment:
Do we need to add the "Alternate" queue in the local `rt_config.yml`?

Reply:
No, the RT client configuration for local dev only needs to specify one queue in order to be functional. I only included an additional (nonexistent) queue in the RT client configuration for automated-testing so that the test suite could test what happens when a configuration file is loaded that contains multiple configured queues.